components/seventodos/seventodos.app.mjs

```diff
@@ -1,11 +1,41 @@
+// The API from 7todo is rejecting the pipedream's axios
+import axios from "axios";
```

Comment:
Can you please start using Pipedream's `axios` lib. More info [here](https://pipedream.com/docs/pipedream-axios/#why-pipedream-platform-axios)

Reply:
Thank you but, as I commented on the line before, the 7todo api doesn't work with Pipedream's Axios, I have tried it several times. 
I always use Pipedream's Axios when it's possible.